commit 3f5080aedd164c1f92a53552dd3e0b82ac6d2bd3
Author: Florian Weimer <fweimer@redhat.com>
Date:   Thu Apr 22 19:53:15 2021 +0200

    nptl: Do not build nptl/tst-pthread-gdb-attach as PIE
    
    (cherry picked from commit 6f3e54d404cfe1ba7d1444e6dfcfd77b102d9287)

diff --git a/nptl/Makefile b/nptl/Makefile
index 1ca23385d4de9000..a1a8ef254b3c3b17 100644
--- a/nptl/Makefile
+++ b/nptl/Makefile
@@ -353,6 +353,9 @@ endif
 CFLAGS-tst-pthread-gdb-attach-static.c := $(CFLAGS-printers-tests)
 CPPFLAGS-tst-pthread-gdb-attach-static.c := \
   $(CFLAGS-printers-tests) -DDO_ADD_SYMBOL_FILE=0
+# As of version 9.2, GDB cannot attach properly to PIE programs that
+# were launched with an explicit ld.so invocation.
+tst-pthread-gdb-attach-no-pie = yes
 
 ifeq ($(build-shared),yes)
 tests-printers-libs := $(shared-thread-library)
